exercises/anagram/anagram.py

```diff
@@ -1,2 +1,2 @@
-def detect_anagrams(string, array):
+def detect_anagrams(string, candidates):
```

Comment:
Your suggestion looks solid, but could you also change `string` to something more meaningful like `word`?

Reply:
Hi @m-a-ge 
i've changed the first parameter to 'word' as your comment. Please review it again :)